Makefile

```diff
@@ -74,7 +76,19 @@ vendors:
 installjs:
 	npm install
 testjs: node_modules
-	@echo "See the tests at http://localhost:8001/umap/static/umap/test/" && python -m http.server 8001
+	@{ \
```

Comment:
For my understanding, can you explain what that trap/wait is doing ? :)

Reply:
Sure,

[`trap`](https://man7.org/linux/man-pages/man1/trap.1p.html) will issue the `kill` command when receiving a signal (here a SIGINT).

[`wait`](https://man7.org/linux/man-pages/man1/wait.1p.html) waits for the job to complete.

To break things down:

- setups a callback with `trap`
- run the server in the background with `&`
- open the browser
- `wait` for the pid to be killed (which will be the case after it's killed with the trap)

Reply:
Ok thanks!
Not sure to understand who/what will send the SIGINT ?

Reply:
`CTRL + C` sends a `SIGINT` 😀 (and `CTRL + D` a `SIGTERM`)

Reply:
It works on my env 👍